src/util/saveArticles.ts

```diff
@@ -348,6 +313,12 @@ export async function saveArticles(zimCreator: ZimCreator, downloader: Downloade
             for (const dep of _moduleDependencies.styleDependenciesList) {
               cssModuleDependencies.add(dep)
             }
+            for (const dep of _moduleDependencies.mobileJsDependenciesList) {
```

Comment:
This is `wikimediaMobileJsDependenciesList`

Reply:
Updated